### ISSUE TITLE
User mass will not increase after death

### ISSUE THREAD

**Issue by aaronwbrown** [mellowDice/ee_server#30]
It seems that after death, the player mass will no longer increment. My assumption is that this is related to our calls to Redis, and how we re-create players after death.

**Comment by TDo13** [comment]
I think it specifically has to do with removing players from the database on death, but not re-adding them on respawn.

**Comment by aaronwbrown** [comment]
Agreed. I think they turn into an anonymous player that has the default mass, but no other properties associated to them.

**Comment by TDo13** [comment]
That and they probably aren't stored in the redis DB at all.  Since the server is in charge of size changing, if the player isn't the redis DB, nothing can happen to it.  I think the fix would be to re-add on re-spawn.
